app/MMB/src/scenes/tickets/boardingPasses/BoardingPassOneWay.js

```diff
@@ -0,0 +1,42 @@
+// @flow strict
+
+import * as React from 'react';
+import { Translation } from '@kiwicom/mobile-localization';
+import { createFragmentContainer, graphql } from '@kiwicom/mobile-relay';
+import { TextIcon, StyleSheet, Color } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+
+import type { BoardingPassOneWay as BookingType } from './__generated__/BoardingPassOneWay.graphql';
+import FlightSegments from './FlightSegments';
+
+type Props = {|
+  +data: BookingType,
+|};
+
+const BoardingPassOneWay = (props: Props) => (
+  <FlightSegments
+    data={idx(props.data, _ => _.trip)}
+    icon={<TextIcon code="&#xe079;" style={styles.outboundIcon} />}
+    iconTitle={
+      <Translation id="mmb.boarding_passes.boarding_pass_return.departure" />
```

Comment:
Should there be a special `id` for OneWay? Or a more generic for all of the types?

Reply:
you are right, good catch